### PR TITLE
Updates to FUSE config files for DR6

### PR DESCRIPTION
**Pull request by jotaylor**
- Add new DR6 FUSE targs to the list of targs to process for HLSPs. 
- Add Van's custom scaled FUSE VO targets
- For Van's custom files, flag all pixels as DQ=3 (rescaled flux)

### CODE REVIEW

**Comment by jotaylor** [comment]
I have tested this code using the code in the feature branch on the DP repo: https://github.com/spacetelescope/ullyses_dp/pull/280
Products are made as expected and look good! So this should be good to merge

File: utils/data/fuse/bad_fuse_targs.yaml [hunk 1]
```diff
@@ -1,22 +1,10 @@
 # For completeness, this is a list of all ULLYSES targets that
 # have FUSE data, but data were not used due to data quality issues.
 # It may be possible to address these issues in the future.
+# This list is not used in any way, just kept for informational purposes.
 - AV83
```
Comment (Rplesha) [inline utils/data/fuse/bad_fuse_targs.yaml:5]: I think that AV83 should also be removed: https://jira.stsci.edu/browse/ULLYSES-1257

Comment (Rplesha) [reply utils/data/fuse/bad_fuse_targs.yaml]: Note- its populated correctly elsewhere.

File: utils/data/fuse/good_fuse_targs.yaml [hunk 2]
```diff
@@ -143,5 +122,41 @@ DR5:
...
+- PGMW-3070
+- SK-67D78
+- SK-69D52
+- SK-70D50
+- SK-71D45
+- BI13
+- SK-66D18
+- SK-66D100
+- SK-67D216
+- SK-70D13
```
Comment (Rplesha) [inline utils/data/fuse/good_fuse_targs.yaml:138]: SK-70D13 on here twice

**Review by Rplesha — COMMENTED** [review]
A couple of minor changes, but otherwise looks good!

File: utils/data/fuse/good_fuse_targs.yaml (hunk 2 above)
Comment (jotaylor) [reply utils/data/fuse/good_fuse_targs.yaml:137]: Fixed!

**Review by Rplesha — APPROVED** [review]

**Pull request merged**